### PR TITLE
Cafuego/ops 6739 mariadb bump

### PR DESCRIPTION
**Pull request by cafuego**

### CODE REVIEW

File: alpine-mysql/unocha.cnf [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+long_query_time                = 5
+
+# I mean, use volumes.
+symbolic_links                 = 0
+
+# Of course.
+skip_name_resolve
+
+# Cache open tables and active threads.
+table_open_cache               = 256
```
Comment (cafuego) [inline alpine-mysql/unocha.cnf:23]: This server only hosts one database, so let's not go overboard.

File: alpine-mysql/unocha.cnf [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+thread_cache_size              = 2
```
Comment (cafuego) [inline alpine-mysql/unocha.cnf:24]: What he said.

File: alpine-mysql/unocha.cnf [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+innodb_log_file_size           = 128M       # This *must* be larger than max_allowed_packet.
+innodb_log_buffer_size         = 8M
+innodb_file_per_table          = 1          # Required for UTF8MB4 support.
+innodb_flush_log_at_trx_commit = 2
+innodb_flush_method            = O_DIRECT   # Bypass the Linux file cache, data is in buffer_pool!
+innodb_lock_wait_timeout       = 50
+innodb_use_native_aio          = 0          # Yeah nah, not in docker.
+
+# This makes changing innodb_log_file_size size safe at any time.
+innodb_fast_shutdown           = 0
```
Comment (cafuego) [inline alpine-mysql/unocha.cnf:42]: This ensures there is *never* uncommitted data in the iblog* transactions logs that is not also in the actual tablespace on shutdown, making it safe to mess with the iblog* files.

File: alpine-mysql/unocha.cnf [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+query_cache_type               = 1
+query_cache_size               = 8M
+query_cache_limit              = 32K
+query_cache_min_res_unit       = 1K
+
+# On local and non-prod this can be fairly low.
+max_connections                = 16
+
+# This should mirror production and existing config, so we can export and import data.
+max_allowed_packet             = 100M
```
Comment (cafuego) [inline alpine-mysql/unocha.cnf:54]: Consistency. It's a thing. 100M is huge, but FTS leaves us no choice.

File: alpine-mysql/Dockerfile.tmpl [hunk 5]
```diff
@@ -20,25 +18,27 @@ LABEL org.label-schema.schema-version="1.0" \
...
+    mv /unocha.cnf /etc/my.cnf.d/unocha.cnf && \
     mkdir -p /etc/fix-attrs.d && \
     cp /fix_data_dir /etc/fix-attrs.d/01-fix-data-dir && \
     cp /fix_logs_dir /etc/fix-attrs.d/02-fix-logs-dir && \
     cp /fix_tmp_dir  /etc/fix-attrs.d/03-fix-tmp-dir
 
 EXPOSE 3306
 
-VOLUME /var/lib/mysql /var/log/mysql /var/tmp/mysql
+VOLUME /var/lib/mysql /var/log/mysql
```
Comment (cafuego) [inline alpine-mysql/Dockerfile.tmpl:44]: No need to make the tmp dir persist unless actually specified by the user.

File: alpine-base-mysql/Dockerfile.tmpl [hunk 6]
```diff
@@ -16,11 +16,11 @@ LABEL org.label-schema.schema-version="1.0" \
       org.label-schema.version=$VERSION \
       org.label-schema.vcs-url=$VCS_URL \
       org.label-schema.vcs-ref=$VCS_REF \
-      org.label-schema.name="alpine-varnish" \
+      org.label-schema.name="alpine-mysql" \
```
Comment (vauxia) [inline alpine-base-mysql/Dockerfile.tmpl:19]: How was this working previously??

File: alpine-mysql/unocha.cnf (hunk 3 above)
Comment (vauxia) [reply alpine-mysql/unocha.cnf:42]: Thank you for your commentary here :)

**Review by vauxia — APPROVED** [review]
Thank you for the helpful docs/commentary :)

**Pull request merged**